constants/TrophyFish.json

```diff
@@ -112,8 +112,8 @@
     },
     "manaray": {
       "displayName": "§9Mana Ray",
-      "description": "§7Caught when you have at least §b1200 ✎ Mana§7.",
-      "rate": 4,
+      "description": "§7Caught when you have at least §b1200 ✎ Mana§7. The catch chance is your §b✎ Mana§7 divided by 1000 (e.g. 1.2% at §b1200 ✎ Mana§7).",
+      "rate": null,
```

Comment:
are you sure setting this to null will not cause bugs in the mod? maybe removing the rate value entirely is better?

Reply:
![image](https://github.com/user-attachments/assets/68fc9919-4ed2-4ba3-995c-46e9e7081151)
Tested In-Game just to check that it will work identically to Golden Fish, Rarity% doesn't Render if Null

Reply:
Yeah, it handles it fine, already used for Golden Fish.